kapitan/refs/functions.py

```diff
@@ -48,6 +49,20 @@ def randomstr(ctx, nbytes=""):
         ctx.data = secrets.token_urlsafe()
 
 
+def randomint(ctx, nbytes=""):
```

Comment:
I see what you're doing here, but just a thought - to me `randomint` should have a behaviour similar to Python `randint()` where it returns a number between its 2 parameters .
Python's `secrets.randbits()` is a bit a different, so maybe call this function `randbits` instead?

Reply:
(or `randbytes`)